core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -203,6 +203,7 @@ angular.module('oppia').directive('tutorCard', [
           ctrl.$onInit = function() {
             $scope.arePreviousResponsesShown = false;
             $scope.waitingForOppiaFeedback = false;
+            $scope.isAudioPlaying = false;
```

Comment:
audioIsPlaying (only start with a verb for methods).

That said, where is this used? I notice that in the directive you use isAudioPlaying(), which suggests that there is another method somewhere with the same name. This seems like a mistake?

Reply:
Removed.